django/website/main/templates/base.html

```diff
@@ -34,7 +34,7 @@ <h1 class="logo"><a href="/">{% settings_value "SITE_NAME" %}</a></h1>
          {% if request.user.preferences.last_viewed_logframe %}
             <div class="navigation-link"><a href="{% url "logframe-dashboard" request.user.preferences.last_viewed_logframe.slug %}" id="nav-dashboard">Dashboard</a></div>
          {% endif %}
-         <div class="account-links-profile"><span>Logged in as: </span><a href="{% url "personal_edit" %}">{{ user.get_full_name|default:user.business_email }}</a></div>
+         <div class="account-links-profile"><span>Logged in as: </span><a href="{% url "personal_edit" %}">{{ request.user.get_full_name|default:user.business_email }}</a></div>
```

Comment:
Isn't this also going to show the wrong email address if request.user,get_full_name is None?